example/test/main.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
It looks to be almost the same as the test added as part of https://github.com/canonical/microcluster/pull/106. 

I am wondering if it would make sense having the test case one instead as it deploys but also runs some checks? 
I have just recently learned about `LXD_INSPECT` in the LXD test suite.
We could introduce something like this so that the deployed MicroCluster can potentially be reused?

Reply:
Yeah, I was planning to rebase #106 onto this but haven't pushed yet.

I'll add `LXD_INSPECT` this afternoon.

Reply:
Totally agree, thanks!